test/functional/page_objects/dashboard_page.js

```diff
@@ -125,24 +125,25 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
      */
     async onDashboardLandingPage() {
       log.debug(`onDashboardLandingPage`);
-      const exists = await testSubjects.exists('dashboardLandingPage');
-      return exists;
+      return await testSubjects.exists('dashboardLandingPage', 5000);
```

Comment:
Looks like I just missed the merge on this; definitely feels cleaner!

I ran the suite a few times locally to try to reproduce the problem, but came up empty as Stacey suspected -- everything worked fine.

I also disabled the above timeout and everything still worked... But of course, that wasn't on CI...